### ISSUE TITLE
Feature requests

### ISSUE THREAD

**Issue by cyberalex4life** [gnunn1/tilix#1182]
Hi, I really love that you've managed to add tabs for sessions.
There are some things I would suggest for the future:
- shortcuts for transparency useful in quake mode for visibility of terminal output or for visibility of a a browser/document window under the terminal;
- resize of tilix quake dimensions (especially height for me) either by shortcuts or, better, by mouse;
- and this one is not a must, but here it goes: show tab bar even if there is only one tab (as an option) for dimension consistency.

**Comment by gnunn1** [comment]
Thanks for the suggestions, here are my thoughts:

1. I don't feel that transparency is adjusted often enough to warrant creating a shortcut for it.

2. You can resize it in the preferences dialog, since the window doesn't have a border it cannot be resized by the mouse.

3. Again I don't think it would be used often to warrant making it an option.

Keep in mind that options have costs in terms of maintenance and testing, I already have too many of them so I'm taking a bit of a harder line these days about adding more.

**Comment by cyberalex4life** [comment]
I would still insist on transparency shortcut, as it is something I often find myself in need due to different colors that make readability terrible, for example this  vagrant dark red error message:
![image](https://user-images.githubusercontent.com/2423189/32663589-2bc1a7ca-c637-11e7-9948-c65189405b3f.png)
It doesn't even exist a shortcut to go  to the profile color each time. I was using this guake feature very often.
This dark red seems to be customized by vagrant as the color scheme has different variants..

**Comment by f2404** [comment]
@cyberalex4life If vagrant indeed doesn't adhere to color scheme, which is weird, selecting the unreadable text with mouse could be a quick solution - in that case the colors would be inverted which could make the text more distinctive.

**Comment by gnunn1** [comment]
I think a better option would be to enable assigning hot keys to profiles so you can switch between profiles quickly and easily. So in your use case, you would have different profiles with colors and transparency set to what you need and switch between the two. This would cover a lot of different profile related items and not just transparency.

Does that make sense?

**Comment by cyberalex4life** [comment]
Well, to tell you the truth, not much, but I've just looked over guake and I think I'm going back to it and I'l try to use it with byobu or something similar. It's probably better for me.
About profiles, in general it complicates things as if you save sessions and every terminal in a session has different profiles, and so on. Overall there's probably too much complexity in tilix for me. And the drop-down feature was added just recently
